pkg/kubelet/nodestatus/setters.go

```diff
@@ -52,6 +53,20 @@ const (
 	MaxNamesPerImageInNodeStatus = 5
 )
 
+func fillNUMANodeCapacity(capacity v1.ResourceList) {
+	info, err := topology.GetNUMANodeInfo()
+
+	if err != nil {
+		klog.Warningf("Can't get NUMANode info: %v", err)
+		return
+	}
+
+	for node, cpuSet := range info {
+		numaKey := fmt.Sprintf("numa%d-cpu", node)
```

Comment:
Rather than prepending with the numa information, I wonder if it might not make sense to have it be the suffix?  There is precedent for this in the linux kernel, which uses a path like "/sys/bus/pci/devices/0000:b3:00.0/numa_node" to report the NUMA node of a PCI device.  This would also let you get the "un-numified" device name by simply dropping the last path segment rather than needing to remove a string from the middle of a path like you have in your suggested numa directory.

Reply:
yes, this approach has number of drawbacks. And we decided to move to CRD https://docs.google.com/document/d/12kj3fK8boNuPNqob6F_pPU9ZTaNEnPGaXEooW1Cilwg/edit.